contingency/contingency-api/src/test/java/com/powsybl/contingency/GeneratorContingencyTest.java

```diff
@@ -19,16 +23,31 @@
 
     @Test
     public void test() {
-        GeneratorContingency contingency = new GeneratorContingency("id");
+        Contingency contingency = Contingency.shuntCompensator("id");
         assertEquals("id", contingency.getId());
-        assertEquals(ContingencyElementType.GENERATOR, contingency.getType());
+        assertEquals(1, contingency.getElements().size());
```

Comment:
This is not equivalent. Is it what you wanted to do? It is strange to find a ShuntContingency in GeneratorContingencyTest

Reply:
My mistake